backend/backend/export.py

```diff
@@ -158,14 +161,30 @@ def float_with_trailing_comma_allowed(s: str) -> float:
         choices=list(gmaps.TravelMode),
         default=gmaps.TravelMode.DRIVE,
     )
+    parser.add_argument(
+        "--departure-time",
+        type=str,
+        default="",
+        help="The time of departure for the route calculation, fromatted as DAY HH(:MM) AM/PM"
```

Comment:
```suggestion
        help="The time of departure for the route calculation, formatted as DAY HH(:MM) AM/PM"
```

Also, could you
- note that this only matters for travel mode TRANSIT
- document the semantics here - i.e. that the actual time is the closest instance of that day-of-week/time combination in the future from the current time